app/src/main/java/com/arygm/quickfix/ui/search/Announcement.kt

```diff
@@ -283,7 +283,10 @@ fun AnnouncementScreen(
 
                     profileViewModel.updateProfile(
                         UserProfile(
-                            currentUserProfile.locations, announcementList, currentUserProfile.uid),
+                            currentUserProfile.locations,
+                            announcementList,
+                            currentUserProfile.wallet,
+                            currentUserProfile.uid),
```

Comment:
The arrival of offline mode and `PreferencesRepositoryDataStore` unfortunately renders this implementation stale and must be updated as soon as the offline mode is up and running. 

I will make sure to add a guide on this on the [repository wiki](https://github.com/arygm/QuickFix/wiki) once that is done.

I am okay with merging this code for now for the sake of proof of concept but an issue must be opened on the subject.

Reply:
I can open an Issue and do that later